unity/Assets/Maroon/scenes/experiments/FallingBallViscosimeter/Scripts/ViscosimeterManager.cs

```diff
@@ -1,185 +1,168 @@
+            }
+        }
+
+        public void OnFluidChange()
+        {
+            _current_fluid = _fluids[_fluid_dropdown.value];
+        }
     
     
-    public decimal calculateDynamicViscosity(decimal temperature)
-    {
-      decimal kelvin = celsiusToKelvin(temperature);
-      return (decimal)(_current_fluid.viscosity_a * Math.Pow(_current_fluid.viscosity_b, (double)kelvin) + _current_fluid.viscosity_c);
-    }
+        public decimal CalculateDynamicViscosity(decimal temperature)
+        {
+            decimal kelvin = CelsiusToKelvin(temperature);
+            return (decimal)(_current_fluid.viscosity_a * Math.Pow(_current_fluid.viscosity_b, (double)kelvin) + _current_fluid.viscosity_c);
+        }
     
-    public decimal celsiusToKelvin(decimal celsius)
-    {
-      return celsius + 273.15m;
-    }
-    void calculateFluidDensity()
-    {
-      //fluid_density_ = (FluidTemperature * -0.37m + 891.83m); //kg/m^3
-      fluid_density_ = (decimal)celsiusToKelvin(FluidTemperature) * (decimal)_current_fluid.density_a + (decimal)_current_fluid.density_b; //kg/m^3
-    }
-
-    public void togglePycnometerFill(bool fill)
-    {
-      if(fill)
-      {
-        pycnometer.fillPycnometer();
-      }
-      else
-      {
-        pycnometer.emptyPycnometer();
-      }
-    }
-
-
-    private void toggleMeasurementBoxes(bool mode)
-    {
-      BroadcastMessage("toggleMeasurement", mode);
-    }
-
-    public void toggleMeasurementMode()
-    {
-      MeasurementMode = !MeasurementMode;
-    }
-
-    void getAllMeasurableObjects()
-    {
-      Object[] foundObjects = Object.FindObjectsOfType<MeasurableObject>();
-
-      measurableObjects = new List<MeasurableObject>();
-
-      foreach (Object obj in foundObjects)
-      {
-        measurableObjects.Add(obj as MeasurableObject);
-      }
-    }
-
-
-    // Start is called before the first frame update
-    void Start()
-    {
+        public decimal CelsiusToKelvin(decimal celsius)
+        {
+            return celsius + 273.15m;
+        }
+        private void CalculateFluidDensity()
+        {
+            fluid_density_ = (decimal)CelsiusToKelvin((decimal)FluidTemperature) * (decimal)_current_fluid.density_a + (decimal)_current_fluid.density_b; //kg/m^3
+        }
+
+        public void TogglePycnometerFill(bool fill)
+        {
+            if(fill)
+            {
+                pycnometer.FillPycnometer();
+            }
+            else
+            {
+                pycnometer.EmptyPycnometer();
+            }
+        }
+
+
+        private void ToggleMeasurementBoxes(bool mode)
+        {
+            BroadcastMessage("toggleMeasurement", mode);
+        }
+
+        public void ToggleMeasurementMode()
+        {
+            MeasurementMode = !MeasurementMode;
+        }
+
+        private void GetAllMeasurableObjects()
+        {
+            Object[] foundObjects = FindObjectsOfType<MeasurableObject>();
```

Comment:
```
           MeasurableObject[] foundObjects = FindObjectsOfType<MeasurableObject>();

            measurableObjects = new List<MeasurableObject>();

            foreach (MeasurableObject obj in foundObjects)
            {
                measurableObjects.Add(obj as MeasurableObject);
            }
```
then we can get rid of using Object = UnityEngine.object at the start of the class